src/globals.d.ts

```diff
@@ -3,3 +3,8 @@ declare function exportFunction(fn: Function, desc: object, out: object): void;
 declare function exportFunction(fn: Function, desc: object): void;
 declare function cloneInto(fn: object, desc: object, out: object): void;
 declare function cloneInto(fn: object, desc: object): void;
+declare namespace  contentScopeFeatures {
```

Comment:
This is just to keep TypeScript happy, since it turns out the `/* global ... */` comment only works for ESLint.